### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Bhashitha19** [learndevopsonline/learndevopsonline#1967] [Q&A]
![Screenshot (65)](https://github.com/user-attachments/assets/8c372131-d7ad-4298-9d28-f4427de5c7f5)

**Comment by Bhashitha19** [comment]
![Screenshot (66)](https://github.com/user-attachments/assets/83bf3a0f-7a71-4d3b-b301-a950668fe1ad)

**Reply by chaithanya1812** [reply]
Can you explain what was the error?

**Reply by Bhashitha19** [reply]
after sending logs to elasticsearch logs are not shown after refresh it throws an error like 
"statusCode": 503,
  "error": "Service Unavailable",
  "message": "License information could not be obtained from Elasticsearch. Please check the logs for further details."

**Reply by chaithanya1812** [reply]
![image](https://github.com/user-attachments/assets/f782a731-556b-472c-b812-b53a09f664d2)

**Reply by Bhashitha19** [reply]
Now please check it once

**Reply by chaithanya1812** [reply]
can you paste your elastisearch password?

**Reply by Bhashitha19** [reply]
elasticsearch password: 0hH7*y-0P0_iyXTcLCsz

**Reply by chaithanya1812** [reply]
![image](https://github.com/user-attachments/assets/3487e78a-bac3-4402-a23b-9d96421a3b4d)

**Reply by Bhashitha19** [reply]
still it does'nt show the logs in elasticsearch

**Reply by chaithanya1812** [reply]
Add the expenses in frontend, then it will show..
For showing logs, first you need to generate the logs..
